### DISCUSSION TITLE
How to open edit/capture in the same window

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#31] [Q&A]
I like to use single windows for my workflow by setting `pop-up-windows` to `nil`, but `ekg` uses `switch-to-buffer-other-window` and forces a split window to open. The irritating part is after hitting `C-c C-c`, I end up with two windows with the same content of `ekg-notes`.

How does one configure `ekg` to continue using the same window.

**Comment by ahyatt** [comment]
Sorry about this, I was dealing with buffers incorrectly.  I switched to what should be a better method in 17ca04af0a16c2461286b71c1843ada4608899c1.  If you update to the latest commit in the `develop` branch, you should get this.

**Comment by jayrajput** [comment]
Thanks for quick fix!!
